### PR TITLE
Update devDependencies

### LINKED ISSUE #194: Storybook CSS looks broken
Looking at the [deployment on now.sh](https://taskforce-fe-components.now.sh/) it seems all the CSS is missing from the components.

### PR DESCRIPTION
**Pull request by Utwo**
### What changed?
Update all devDependencies. Close #194

### Actions (optional)
 - [x] Increase the version of the package if you need to release it after the merge. This will bump to version 2.0.0
 - [ ] If you added a new component, please make sure to export it in `../src/index.js`

### CODE REVIEW

**Comment by surdu** [comment]
So... many... commas ... 😝

**Comment by Utwo** [comment]
> So... many... commas ... stuck_out_tongue_closed_eyes

I think it's because of the eslint and prettier update. They changed the recommendation.

**Comment by surdu** [comment]
All those commas makes it impossible to review properly :( Can't we disable that rule? The rule is also not beneficial in my opinion.

File: src/components/sidebar-menu-item/sidebar-menu-item.stories.js [hunk 1]
```diff
@@ -27,25 +26,25 @@ export const withTheme = () => (
   <>
     <SidebarMenuItem
       isTitle
-      theme={{ highlightColor: "#F6DD62", backgroundColor: "#F6F9FC" }}
+      style={{ "--highlightColor": "#F6DD62", "--backgroundColor": "#F6F9FC" }}
```
Comment (surdu) [inline src/components/sidebar-menu-item/sidebar-menu-item.stories.js:29]: Is the "--" prefix a react thing or am I missing something ?

Comment (Utwo) [reply src/components/sidebar-menu-item/sidebar-menu-item.stories.js:29]: No, it's a CSS variable. Basically with style prop you can send any CSS. In the example above, I added to CSS variable that will change the color of the sidebar menu.

Comment (cipick) [reply src/components/sidebar-menu-item/sidebar-menu-item.stories.js:29]: The is a breaking change and we should have a note for projects which are using this component to update the props

**Comment by cipick** [comment]
Looks good for me

File: src/components/sidebar-menu-item/sidebar-menu-item.stories.js (hunk 1 above)
Comment (Utwo) [reply src/components/sidebar-menu-item/sidebar-menu-item.stories.js:29]: After merge I will take care on updating this on all projects that depend on fe-components.

Comment (surdu) [reply src/components/sidebar-menu-item/sidebar-menu-item.stories.js:29]: After merging this maybe we should also merge [this](https://github.com/code4romania/taskforce-fe-components/pull/150) and then update projects and release notes

Comment (Utwo) [reply src/components/sidebar-menu-item/sidebar-menu-item.stories.js:29]: definitely

**Pull request merged**